templates/ohdsi-webapi/atlas_database.bicep

```diff
@@ -0,0 +1,147 @@
+param location string
+param suffix string
+param odhsiWebApiName string
+param branchName string = 'v2'
```

Comment:
`branchName` shouldn't be defined also in `main.bicep`?  otherwise, how can you override it when deploying main?
If you agree please move the default assignment to `main.bicep`

Reply:
Agree, done